src/test/java/edu/harvard/hul/ois/fits/junit/service/DocMDXmlUnitServiceTest.java

```diff
@@ -38,338 +31,89 @@
  * @author dan179
  */
 @Ignore
```

Comment:
Are we ignoring these tests because they do not work?

Reply:
@awoods I believe all of the tests that you noted are ignored because they require a FITS web service to be running external to the tests. I have no idea if they do or don't work.